tests/queries/0_stateless/03761_inconsistent_ast_formatting.sql

```diff
@@ -0,0 +1,2 @@
+select (-(42))[3]; -- {serverError ILLEGAL_TYPE_OF_ARGUMENT}
+select(-('a')).1; -- {serverError ILLEGAL_TYPE_OF_ARGUMENT}
```

Comment:
Let's convert the test to `explain syntax`, to fix bug check and make it more explicit